.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: java
+
+sudo: false
+
+jdk:
+  - oraclejdk8
```

Comment:
Should we perhaps use higher java version? There is outstanding PR for supporting java 11...

Reply:
Sure, we can do that.  In this case, we'd have to wait for Java 11 support to be added before using it.

We can also run tests with each JDK.

In either case, I think we should still run tests against JDK 8, so we should merge this PR and have another one to add a JDK 11 build.